.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,92 @@
+name: "🐛 Bug report"
+description: "Something not working the way it should be?"
+
+title: "[Bug?]: "
+labels: bug/needs-info
+
+body:
+  - type: textarea
+    attributes:
+      label: What's not working?
+      description: >
+        Describe the bug here.
+        The more detail you provide, the better,
+        but try to start with a summary to frame the issue.
+      placeholder: >
+        The dev server doesn't seem to be working.
```

Comment:
This stuff really is too wordy, no? 

I definitely would take out the suggestion about bumping webpack, because guesses like this unless linked with a specific issue are probably more harmful than helpful

Reply:
The motivation here was to provide people opening issues with an example of what a good one looks like, but instead of including it here, we can just link out to it after we have a good one written up with the new template.